action.yml

```
@@ -6,6 +6,12 @@ branding:
   color: purple
 description: Setup a shared GitHub Actions workflow environment.
 outputs:
+  go-junit-report-version:
+    description: The version of go junit report to use.
+    value: ${{ steps.go-junit-report.outputs.version }}
+  go-mock-version:
+    description: The version of go mock to use.
```

Clarify which repos are being versioned:
```suggestion
    description: The version of jstemmer/go-junit-report to use.
    value: ${{ steps.go-junit-report.outputs.version }}
  go-mock-version:
    description: The version of golang/mock to use.
```